web/src/app/main/components/help/help.component.ts

```diff
@@ -19,65 +17,16 @@ export class HelpComponent implements OnInit {
 
   public searchForm: FormGroup;
   public filteredTopics: Help[] = [];
-  public topics: Help[] = [
-    {
-      title: 'Quickstart',
-      description: 'Overview information on getting started',
-      icon: 'quickstart_icon',
-      path: 'quickstart',
-      updatedAt: '09/13/2019 09:05 AM',
-    },
-    {
-      title: 'Glossary',
-      description: 'Coming Soon',
-      icon: 'glosary_icon',
-      path: 'coming-soon',
-      updatedAt: '09/13/2019 09:05 AM',
-    },
-    {
-      title: 'Create Project',
-      description: 'Coming Soon',
-      icon: 'create_enviroment_icon',
-      path: 'coming-soon',
-      updatedAt: '09/13/2019 09:05 AM',
-    },
-    {
-      title: 'Edit Project',
-      description: 'Coming Soon',
-      icon: 'edit_enviroment_icon',
-      path: 'coming-soon',
-      updatedAt: '09/13/2019 09:05 AM',
-    },
-    {
-      title: 'Delete Project',
-      description: 'Coming Soon',
-      icon: 'delete_enviroment_icon',
-      path: 'coming-soon',
-      updatedAt: '09/13/2019 09:05 AM',
-    },
-    {
-      title: 'How to support us',
-      description: 'Coming Soon',
-      icon: 'how_icon',
-      path: 'coming-soon',
-      updatedAt: '09/13/2019 09:05 AM',
-    },
-    {
-      title: 'Why Open Source?',
-      description: 'Coming Soon',
-      icon: 'why_icon',
-      path: 'coming-soon',
-      updatedAt: '09/13/2019 09:05 AM',
-    },
-  ];
+  public help: HelpModel = new HelpModel();
+  public topics: Help[];
 
   constructor(
     private http: HttpClient,
-    private form: FormBuilder,
-    private dialog: MatDialog
+    private form: FormBuilder
   ) { }
 
   ngOnInit(): void {
+    this.topics = this.help.topics;
     this.topics.forEach((help: { title: string, path: string }, index: number) => {
       this.http.get(`/assets/help/${help.path}.md`, { responseType: 'text' }).subscribe((content: string) => {
```

Comment:
I know this was there already, but might be cleaner to do it as a resolve?

Reply:
okay